{"uid":"be6ef5bb554cf01f","name":"test_pre_upgrade_check","fullName":"tests.integration.ha_tests.test_upgrade_from_stable#test_pre_upgrade_check","historyId":"6f73cd86130887313b53a62d2e64791e","time":{"start":1713750401071,"stop":1713750401071,"duration":0},"description":"Test that the pre-upgrade-check action runs successfully.","descriptionHtml":"<p>Test that the pre-upgrade-check action runs successfully.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 158, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 515, in setup\n    raise exc\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 512, in setup\n    col.setup()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1775, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 687, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 541, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 570, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 647, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1079, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1713749195822,"stop":1713749195822,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1713749195823,"stop":1713749195953,"duration":130},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1713749195823,"stop":1713749195823,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1713750401071,"stop":1713750401071,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1713750401075,"stop":1713750401338,"duration":263},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1713750401338,"stop":1713750401339,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1713750401339,"stop":1713750401339,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1713750401339,"stop":1713750401339,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_upgrade_from_stable"},{"name":"host","value":"fv-az564-723"},{"name":"thread","value":"8350-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_upgrade_from_stable"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":9,"unknown":0,"total":11},"items":[{"uid":"3d9eab3d087c1b61","reportUrl":"../18//#testresult/3d9eab3d087c1b61","status":"passed","time":{"start":1713663576818,"stop":1713663579202,"duration":2384}},{"uid":"19cafd9e960752c1","reportUrl":"../16//#testresult/19cafd9e960752c1","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1713577607768,"stop":1713577607768,"duration":0}},{"uid":"c1b811acdcd9e84d","reportUrl":"../15//#testresult/c1b811acdcd9e84d","status":"passed","time":{"start":1713490783103,"stop":1713490785443,"duration":2340}},{"uid":"ea2557046052eab4","reportUrl":"../14//#testresult/ea2557046052eab4","status":"passed","time":{"start":1713404322664,"stop":1713404326007,"duration":3343}},{"uid":"eb4a58af0e2ee128","reportUrl":"../13//#testresult/eb4a58af0e2ee128","status":"passed","time":{"start":1713318032580,"stop":1713318036003,"duration":3423}},{"uid":"3b49c8005890c3f0","reportUrl":"../8//#testresult/3b49c8005890c3f0","status":"passed","time":{"start":1713231675420,"stop":1713231678707,"duration":3287}},{"uid":"fb9bb25d37bfd295","reportUrl":"../7//#testresult/fb9bb25d37bfd295","status":"passed","time":{"start":1713150544997,"stop":1713150548262,"duration":3265}},{"uid":"f7aed6620845bd83","reportUrl":"../6//#testresult/f7aed6620845bd83","status":"passed","time":{"start":1713058902204,"stop":1713058904501,"duration":2297}},{"uid":"1cf08472ccbd4a79","reportUrl":"../4//#testresult/1cf08472ccbd4a79","status":"passed","time":{"start":1712972080849,"stop":1712972084114,"duration":3265}},{"uid":"ec2f1004abe07fb1","reportUrl":"../2//#testresult/ec2f1004abe07fb1","status":"passed","time":{"start":1712886020259,"stop":1712886022564,"duration":2305}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"be6ef5bb554cf01f.json","parameterValues":[]}